### PR TITLE
add JSON test

### PR DESCRIPTION
**Pull request by CJentzsch**
* add JSON test blockchaintest in order to test JSON RPC @frozeman

### CODE REVIEW

File: test/JSON_test.sol [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    bool myBool;
+    int8 myInt8;
+    uint8 myUint8;
+    int256 myInt256;
+    uint256 myUint256;
+    address myAddress;
+    bytes0 myBytes0;
+    bytes32 myBytes32;
+
```
Comment (winsvega) [inline test/JSON_test.sol]: empty line?

**Comment by gavofyork** [comment]
rebase?

**Comment by gavofyork** [comment]
fails on windows?

**Comment by winsvega** [comment]
this seems to be an old issue.

**Comment by bobsummerwill** [comment]
It's a merge from April 2015, @winsvega.   Not sure why you are commenting on it :-)

**Comment by winsvega** [comment]
cause it is in "in progress"  column

**Comment by bobsummerwill** [comment]
I removed that tag.

**Pull request merged**